### PR TITLE
chore: move ui back into monorepo

### PR DESCRIPTION
**Pull request by markphelps**
Fixes: FLI-219

- Moves UI code back to this repo under `ui` directory
- Fix CI / dagger to use local `ui` folder instead of cloning from other repo

All of the code in `ui` is a direct copy over from the https://github.com/flipt-io/flipt-ui repo as of commit [`f968a91`](https://github.com/flipt-io/flipt-ui/commit/f968a91d5bcde81078a1ccd5c37c757c0378bcde)

## TODO

(in follow up PRs)

- Rethink/Redo development workflow now that UI is back in this repo
- Add back UI ITs to build/run from this repo instead of relying on `flipt/flipt:nightly` image

### CODE REVIEW

File: ui/dev.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ package ui
 import "embed"
 
 var (
-	//go:embed index.html
+	//go:embed dev.html
```
Comment (markphelps) [inline ui/dev.go:9]: TODO: this doesnt work yet. will  have to redo the development workflow in another PR

File: ui/dev.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```
Comment (markphelps) [inline ui/dev.html:1]: same as above, this is just copied over for posterity

**Review by GeorgeMac — APPROVED** [review]
Welcome home UI 🏡

**Comment by markphelps** [comment]
@all-contributors please add @darinmclain  for code

**Pull request merged**